app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -19,12 +24,22 @@ type WishlistOutput @doc(description: "Deprecated: `Wishlist` type should be use
 
 type Wishlist {
     id: ID @doc(description: "Wishlist unique identifier")
-    items: [WishlistItem] @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistItemsResolver") @doc(description: "An array of items in the customer's wish list"),
-    items_count: Int @doc(description: "The number of items in the wish list"),
-    sharing_code: String @doc(description: "An encrypted code that Magento uses to link to the wish list"),
+    items: [WishlistItem] @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistItemsResolver") @deprecated(description: "Use field `items_v2` from type `Wishlist` instead")
```

Comment:
Shouldn't this be @deprecated(reason: ) ?

Reply:
Definitely should be. 
Thank you for spotting this one.